### PR TITLE
DEX-973 Order history route tests

### PR DESCRIPTION
**Pull request by ngadiyak**

### CODE REVIEW

File: dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpSuccessfulDeleteHistory.scala [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import com.wavesplatform.dex.domain.order.Order
+import io.swagger.annotations.{ApiModel, ApiModelProperty}
+import play.api.libs.json.{Json, OFormat}
+
+
+@ApiModel(
+  description = "Successful deletion message. Can be HttpSuccessfulDeleteHistory",
+  subTypes = Array(
+    classOf[HttpSuccessfulSingleCancel],
+    classOf[HttpSuccessfulBatchCancel]
```
Comment (vsuharnikov) [inline dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpSuccessfulDeleteHistory.scala:12]: It looks like the `subTypes` are incorrect. Do they?

File: dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpSuccessfulDeleteHistory.scala [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  )
+)
+class HttpSuccessfulDelete {
```
Comment (vsuharnikov) [inline dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpSuccessfulDeleteHistory.scala:15]: If we have such class, we could use it in Swagger docs? Now I see:
```scala
response = classOf[HttpMessage]
```
for `def deleteHistory`

**Review by vsuharnikov — CHANGES_REQUESTED** [review]

File: dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpSuccessfulDeleteHistory.scala (hunk 1 above)
Comment (ngadiyak) [reply dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpSuccessfulDeleteHistory.scala:12]: incorrect, yes

File: dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpSuccessfulDeleteHistory.scala (hunk 2 above)
Comment (ngadiyak) [reply dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpSuccessfulDeleteHistory.scala:15]: ok

**Review by vsuharnikov — APPROVED** [review]

**Pull request merged**